source/includes/_documentation.md

```diff
@@ -4854,6 +4854,194 @@ The angular Button:
+@UI("#list-with-sections mat-list-item") 
+public JList<Label> listWithSection; 
+
+@Test 
+public void basicListBasicTest() { 
+    listSection.basicList.is().displayed(); 
+} 
+
+@Test 
+public void basicListTextTest() { 
+    listSection.basicList.get(1).is().text("Item 1"); 
+}
+
+@Test
+public void listWithSectionsBasicTest() {
+    listSection.listWithSection.is().displayed();
+}
+
+@Test
+public void listWithSectionsIconTest() {
+    listSection.listWithSection.get(1).children().get(3).is().text("folder");
+    listSection.listWithSection.get(5).children().get(3).is().text("note");
+}
+
+@Test 
+public void listWithSectionsTextTest() { 
+    listSection.listWithSection.get(1).is().text("folder\nPhotos\nJan 1, 2016"); 
+} 
+
+``` 
+
+```html 
+<mat-list id="basic-list" role="list"> 
+<mat-list-item id="basic-list-item-1" role="listitem">Item 1</mat-list-item> 
+<mat-list-item id="basic-list-item-2" role="listitem">Item 2</mat-list-item> 
+<mat-list-item id="basic-list-item-3" role="listitem">Item 3</mat-list-item> 
+</mat-list> 
+``` 
+
+To add an icon to your list item, use the matListIcon attribute. 
+Subheader can be added to a list by annotating a heading tag with an matSubheader attribute. To add a divider, use <mat-divider>. 
+
+See an example with HTML code describing list with sections element. 
+
+![List examples](../images/angular/list_with_sections.png) 
+
+```html 
+<mat-list id="list-with-sections"> 
+<div mat-subheader>Folders</div> 
+<mat-list-item id="{{ 'list-with-section-items-' + folder.name.toLowerCase() }}" *ngFor="let folder of folders"> 
+<mat-icon mat-list-icon>folder</mat-icon> 
+<div mat-line>{{folder.name}}</div> 
+<div mat-line> {{folder.updated | date}} </div> 
+</mat-list-item> 
+<mat-divider></mat-divider> 
+<div mat-subheader>Notes</div> 
+<mat-list-item id="{{ 'list-with-section-items-' + note.name.toLowerCase().replace(' ','-') }}" *ngFor="let note of notes"> 
+<mat-icon mat-list-icon>note</mat-icon> 
+<div mat-line>{{note.name}}</div> 
+<div mat-line> {{note.updated | date}} </div> 
+</mat-list-item> 
+</mat-list> 
+``` 
+
+|Method | Description | Return Type 
+--- | --- | --- 
+**is()** | Assert action | TextAssert 
+**displayed()** | Check that element is displayed | TextAssert 
+**get(int index)** | Select label by index | Label 
+**show()** | Scroll to element | void 
+**text(String text)** | Check whether a text matches a pattern | TextAssert 
+
+#### <a href="https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ListTests.java" target="_blank">List java tests examples</a>
+### Grid list 
+
+#### <a href="https://material.angular.io/components/grid-list/overview" target="_blank">Grid list overview</a>
+
+Grid list is a two-dimensional list view that arranges cells into grid-based layout. 
+
+There is two different grid lists in Angular: Basic grid list and Dynamic grid list. 
```

Comment:
There seems to be a typo: "There is" -> "There are"

Reply:
Yes. Changes added.